src/lib/rest/OrionError.cpp

```diff
@@ -109,7 +109,7 @@ std::string OrionError::smartRender(ApiVersion apiVersion)
 */
 std::string OrionError::setStatusCodeAndSmartRender(ApiVersion apiVersion, HttpStatusCode* scP)
 {
-  if (apiVersion == V2)
+  if ((apiVersion == V2) || (apiVersion == ADMIN_API))
```

Comment:
Great to see that the addition has not broken anything... setStatusCodeAndSmartRender() modifyication were hard in the past (at least to me...)

NTC